Radiant/fileHandler.js

```diff
@@ -112,43 +108,33 @@ const checkM3U8 = (file) => {
 /**
  * checkFileAgain
  * @param info
+ * @param previousSize
  */
-const checkFile = function (info){
+const checkFile = function (info, previousSize){
     const ext = info.path.replace(/^.*[\\\/]/, '').split('.')[1];
     if(ext === 'm3u8'){
-        uploadFile(info);
+        uploadFile(info, false);
     } else {
-        setTimeout((args) => {
-            fs.stat(args[0].info.path, (err, fileInfo) => {
-                if(err === null) {
-                    if(fileInfo.size <= 50000) {
-                        // console.log(`-=*[ checking file: ${info.path} with size: ${fileInfo.size}: checking again in 1.5 sec ]*=-`);
-                        if(streamTracker[info.path].retry <= 4){
-                            streamTracker[info.path].retry++;
-                            checkFile(info);
-                        } else {
-                            uploadFile(info);
-                        }
-                    } else {
-                        delete streamTracker[info.path];
-                        uploadFile(info);
-                        // console.log(`-=*[ uploading file: ${info.path} with size: ${fileInfo.size} ]*=-`);
-                    }
-                }  else {
-                    console.log(`File not found ${err}`);
+        fs.stat(info.path, (err, fileInfo) => {
+            if(err === null) {
+                if(fileInfo.size === previousSize && fileInfo.size > 0) {
+                    uploadFile(info, false);
+                } else {
+                    checkFile(info, fileInfo.size);
                 }
-            });
-        }, 1000, [{
-            info
-        }]);
+            }  else {
+                console.log(`File not found ${err}`);
```

Comment:
is this an error state we should actually keep track of?

Reply:
no